packages/core/src/capture/storage-manager.ts

```diff
@@ -166,6 +177,129 @@ export class StorageManager {
     return await backend.getSessions(serverName);
   }
 
+  /**
+   * Get client aggregations from the first available backend
+   *
+   * @returns List of client aggregation info
+   */
+  async getClients(): Promise<ClientAggregation[]> {
+    if (!this.initialized) {
+      throw new Error(
+        "Storage manager not initialized. Call initialize() first.",
+      );
+    }
+
+    const backend = this.backends.values().next().value as
+      | StorageBackend
+      | undefined;
+    if (!backend) {
+      throw new Error("No storage backends registered");
+    }
+
+    return await backend.getClients();
+  }
+
+  /**
```

Comment:
this references "all backends" - just making sure this is not a remnant from the old approach where we wanted to back both sqlite and fs